CMakeLists.txt

```diff
@@ -199,9 +194,9 @@ include(InstallBasicPackageFiles)
 # SCENARI/O
 # =========
 
-option(GYMIGNITION_ENABLE_SCENARIO "Enable ScenarI/O library" ON)
+option(SCENARIO_ENABLE_SCENARIO "Enable ScenarI/O library" ON)
```

Comment:
This option has a strange name now, did you noticed?

Reply:
Nope :) Good catch, I think this should be enabled by default.

Reply:
Done in https://github.com/robotology/gym-ignition/compare/877b753b1300d4ee2bffb67a338fa4399b972517..7b99f6b70e4686df0843e666392f4b248b970741